lib/Elastica/Aggregation/Filters.php

```diff
@@ -22,10 +22,12 @@ class Filters extends AbstractAggregation
      */
     public function addFilter(AbstractFilter $filter, $name = '')
     {
-        if (empty($name)) {
-            $filterArray[] = $filter->toArray();
+        $filterArray = array();
+
+        if (is_string($name)) {
```

Comment:
@ruflin change that @stof notice I made to sync this check between toArray and addFilter, because they must be same. I think additionally to https://github.com/ruflin/Elastica/commit/5a08b5f66b76f2f91ecba6213a7a6b9c9225326d there is must be changed addFilter method in same way. What do you think? Or I was wrong when say that check must be same between this two methods.

This change came from https://github.com/ruflin/Elastica/pull/916#discussion-diff-37099377